tests/compile_tests/rotations/test.output

```diff
@@ -0,0 +1,20 @@
+ 240 :11110000
```

Comment:
I really like the output formatting :) Makes it pretty easy to verify by eye.

I think it would be good to add a few more test cases for signed integer variables, and also rotations that are negative and rotations that rotate past the size of the integer (Ex. `_ROR(a, -1)`, `_ROR(a, 8)`, `_ROR(a, 17)`).